sdk/storage/storage-file-share/src/generated/src/models/parameters.ts

```diff
@@ -349,7 +349,7 @@ export const fileLastWriteOn: coreHttp.OperationParameter = {
     required: true,
     serializedName: "x-ms-file-last-write-time",
     type: {
-      name: "String"
+      name: "DateTimeRfc1123"
```

Comment:
Should this be reverted? (with a transform)

Reply:
core-http tries to validate and throws an error as follows.
![image](https://user-images.githubusercontent.com/10452642/67994729-f9f20500-fc03-11e9-8f8b-d6a0555bed3c.png)


![image](https://user-images.githubusercontent.com/10452642/67994723-eb0b5280-fc03-11e9-818a-77b04ec709a2.png)

Reply:
@XiaoningLiu

Reply:
Reverted this with a transformation in the swagger readme, for now, to get the existing tests to succeed.

Reply:
Python sdk is following a similar [workaround](https://github.com/Azure/azure-sdk-for-python/tree/azure-storage-file-share_12.0.0b5/sdk/storage/azure-storage-file-share/swagger#change-new-smb-file-parameters-to-use-default-values).